keanu-python/keanu/keanu_random.py

```diff
@@ -22,3 +23,8 @@ def next_double(self) -> float:
     @staticmethod
     def set_default_random_seed(seed: int) -> None:
         k.jvm_view().KeanuRandom.setDefaultRandomSeed(seed)
+
+
+def set_deterministic_state():
```

Comment:
should this be a static method?

Reply:
I personally think it's better just being a function given it's not specifically related to the KeanuRandom object - it's just something that stops things being non-deterministic and interfaces with the KeanuRandom class to do that.  Happy to hear more thoughts though.